packages/dx-demo-shell/src/utils.js

```diff
@@ -0,0 +1,6 @@
+export const findOccurrence = (source, strings) => {
+  for (let index = 0; index < strings.length; index += 1) {
+    if (source.indexOf(strings[index]) > -1) return true;
+  }
+  return false;
+};
```

Comment:
[Array.find()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) ?

Reply:
Do you suggest use `array.find()` instead of `array.findIndex()`?

Reply:
Maybe `strings.some(s => source.indexOf(s) > -1)` ?

Reply:
`Array.includes() `?